src/test/com/abstract_calss/Animal.java

```diff
@@ -0,0 +1,7 @@
+package test.com.abstract_calss;
+
+public abstract class Animal {
+    abstract void move();
```

Comment:
empty line after class is a part of a convention, it's a must follow convention. You shouldn't do such mistakes neither at work nor during education.